confignode/src/main/java/org/apache/iotdb/confignode/procedure/env/DataNodeRemoveHandler.java

```diff
@@ -207,6 +207,8 @@ public void updateRegionLocationCache(
         status,
         originalDataNode.getInternalEndPoint().getIp(),
         destDataNode.getInternalEndPoint().getIp());
+    // Broadcast the latest RegionRouteMap when Region migration finished
+    configManager.getLoadManager().broadcastLatestRegionRouteMap();
```

Comment:
The result is irrelevant with the Region migration since the update cache step happened after the physical migration succeed. Therefore, no matter what the result of updateRegionLocation is, we should process a broadcast.